tljh/requirements-base.txt

```diff
@@ -5,8 +5,10 @@
 jupyterhub==1.2.*
 notebook==6.1.*
 # Install additional notebook frontends!
-jupyterlab==2.*
+jupyterlab==3.*
 nteract-on-jupyter==2.1.*
+# Install jupyterlab extensions from PyPI
+jupyterlab-widgets==1.*
 # nbgitpuller for easily pulling in Git repositories
 nbgitpuller==0.9.*
 # nbresuse to show people how much RAM they are using
```

Comment:
And `jupyter-resource-usage==0.5.*` now includes the prebuilt extension JupyterLab 3.0.

Reply:
We might also want to update to `tornado>=6.1` since this is what JupyterLab 3.0 requires:

https://github.com/jupyterlab/jupyterlab/blob/be28462667a9a6617313ba33818559bdd9cf4d9c/setup.py#L147

Reply:
About tornado: yep, otherwise it gives an error:

```
root@jupytest:~# /opt/tljh/user/bin/jupyter lab build --minimize=False
Traceback (most recent call last):
  File "/opt/tljh/user/lib/python3.7/site-packages/jupyter_server/serverapp.py", line 54, in <module>
    assert tornado.version_info >= MIN_TORNADO
AssertionError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/opt/tljh/user/bin/jupyter-lab", line 6, in <module>
    from jupyterlab.labapp import main
  File "/opt/tljh/user/lib/python3.7/site-packages/jupyterlab/__init__.py", line 7, in <module>
    from .labapp import LabApp
  File "/opt/tljh/user/lib/python3.7/site-packages/jupyterlab/labapp.py", line 16, in <module>
    from jupyterlab_server import slugify, WORKSPACE_EXTENSION
  File "/opt/tljh/user/lib/python3.7/site-packages/jupyterlab_server/__init__.py", line 4, in <module>
    from .app import LabServerApp
  File "/opt/tljh/user/lib/python3.7/site-packages/jupyterlab_server/app.py", line 11, in <module>
    from jupyter_server.extension.application import ExtensionApp, ExtensionAppJinjaMixin
  File "/opt/tljh/user/lib/python3.7/site-packages/jupyter_server/extension/application.py", line 21, in <module>
    from jupyter_server.serverapp import ServerApp
  File "/opt/tljh/user/lib/python3.7/site-packages/jupyter_server/serverapp.py", line 58, in <module>
    ) from e
ImportError: The Jupyter Server requires tornado >=6.1.0
```